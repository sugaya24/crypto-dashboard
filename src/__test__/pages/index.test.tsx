import React from 'react';
import Home from '../../pages';
import { render } from '../utils';

describe(`Home`, () => {
  it(`show home page`, () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
