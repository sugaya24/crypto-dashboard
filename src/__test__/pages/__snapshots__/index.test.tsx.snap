// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home show home page 1`] = `
<div>
  <div>
    <main>
      <h2
        class="chakra-heading css-1dklj6k"
      >
        Crypto Dashboard
      </h2>
    </main>
  </div>
</div>
`;
